Cargo.toml

```diff
@@ -36,3 +36,6 @@ harness = false
 [badges]
 travis-ci = { repository = "hmeyer/tessellation", branch = "master" }
 codecov = { repository = "hmeyer/tessellation", branch = "master", service = "github" }
+
+[patch.crates-io]
+bbox = { git = "https://github.com/benmkw/bbox" }
```

Comment:
I just cherry-picked your 6ae2799.
So I guess we can just dep on bbox = "0.10" ?

Reply:
cool, thanks, sure!

I could use some help with updating nalgebra if you have time, it seems they are in disagreement about their traits a bit, maybe I should write up the problem more accurately but I basically tried to resolve it for two hours and failed :D

Reply:
seems like we need to bump it in implicit3d as well, should I do the same changes there as well? sorry these changes spread so much :/

Reply:
Yes, updating nalgebra is a pain.
I usually add a minor version bump for all libs when updating nalgebra.

Reply:
I updated bbox, tessellation and implicit3d to nalgebra 0.22.

Reply:
yeah but it still does not work for tesselation :D at least I think it may be easier for you to apply the changes but I can look into it as well if you prefer just need some more time :)

Reply:
ok so i got it compiling but sometimes there are now 3 different RealField requirements which seems a bit odd, maybe I can remove some and then its fine again

Reply:
it seems a bit odd that tesselation now depends on traits from implicit3d rather than alga and nalgebra as this defeats the purpose of generics a bit as one now has to depend on implciit3d just to get the traits if I understand correctly. 

Maybe it would thus be better to use the actual two traits even though thats noisier.